build-system/tasks/extension-generator/extension-doc.template.md

```diff
@@ -0,0 +1,220 @@
+---
```

Comment:
At Alan's request, this file was removed and the regular/legacy AMP extension generator reads from the file above. I believe the thought behind it was to ensure they stay in sync, and the `.md` is shared between versions anyway. IIUC, this file can be removed, and any changes should be reflected in the one above.

Reply:
Thanks! I've moved the changes over - please let me know if I've done anything incorrectly or misunderstood.